akka-actor/src/main/scala/akka/actor/dungeon/Children.scala

```diff
@@ -23,7 +24,7 @@ private[akka] trait Children { this: ActorCell =>
 
   import ChildrenContainer._
 
-  @nowarn("msg=never used")
+  @nowarn("msg=never")
```

Comment:
To work both in 2.12 and 2.13, or should this be just `@nowarn`?

Reply:
Good to use explicit message in case something else comes up later. Shorter `never` is fine.

Reply:
Yeah, it's `never updated` now and in 2.12 it's `never used`.